searx/engines/brave.py

```diff
@@ -0,0 +1,250 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""
+ Brave (General, news, videos, images)
+"""
+# pylint: disable=fixme
+
+from urllib.parse import (
+    urlencode,
+    urlparse,
+    parse_qs,
+)
+
+import chompjs
+from lxml import html
+
+from searx.utils import (
+    extract_text,
+    eval_xpath_list,
+    eval_xpath_getindex,
+)
+
+about = {
+    "website": 'https://search.brave.com/',
+    "wikidata_id": 'Q22906900',
+    "official_api_documentation": None,
+    "use_official_api": False,
+    "require_api_key": False,
+    "results": 'HTML',
+}
+
+base_url = "https://search.brave.com/"
+categories = []
+brave_category = 'search'  # search, images, videos, news
+brave_spellcheck = False
+
+send_accept_language_header = True
+paging = False
+"""By Brave, paging is only supported in :py:obj:`brave_category`
```

Comment:
I think this should be `At Brave`, or `Brave only supports paging in ...`, `By Brave` sounds very German :P